convert_hf_to_gguf.py

```diff
@@ -3022,14 +3028,30 @@ def tensor_force_quant(self, name, new_name, bid, n_dims):
             return gguf.GGMLQuantizationType.F32
         return False
 
+    def _mapping_interns1_name(self, name):
+        names_map = {
+            "model.multi_modal_projector.layer_norm.bias": "mlp1.0.bias",
+            "model.multi_modal_projector.layer_norm.weight": "mlp1.0.weight",
+            "model.multi_modal_projector.linear_1.bias": "mlp1.1.bias",
+            "model.multi_modal_projector.linear_1.weight": "mlp1.1.weight",
+            "model.multi_modal_projector.linear_2.bias": "mlp1.3.bias",
+            "model.multi_modal_projector.linear_2.weight": "mlp1.3.weight",
+        }
```

Comment:
Hmm ok I think the mapping of 6 tensors can't be added to `tensor_mapping.py`, as it will mess up conversion for other models. So it's ok to keep these 6 tensors here for now.

But one thing I'm not sure, why mapped name you are using is `mlp%d.%d.%s`? I think it should be `mm.model.mlp.%d.%s` to match the original InternVL model

Reply:
it just maps interns1 weight name to internvl weight name
https://github.com/ggml-org/llama.cpp/blob/00131d6eaf4df029e1ec84de868c2c5957503007/gguf-py/gguf/tensor_mapping.py#L1078